oidc-handler/pom.xml

```diff
@@ -1,11 +1,35 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.sling</groupId>
+  <artifactId>org.apache.sling.oidchandler</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <name>org.apache.sling.oidchandler</name>
 
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <properties>
+    <sling.username>admin</sling.username>
+    <sling.password>admin</sling.password>
+  </properties>
 
-    <modelVersion>4.0.0</modelVersion>
-    <groupId>org.apache.sling</groupId>
-    <artifactId>org.apache.sling.oidchandler</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <build>
```

Comment:
You won't the build section anymore, this will be just a 'reactor' project.

Reply:
If this plugin is removed, the project cannot be implemented using mvn package sling:install.

Reply:
True, you should:

- remove this
- update the other modules to inherit from the sling parent pom

Do you want to run `mvn package sling:install` from the reactor? Then you might need to also add the sling parent to this pom. But I suggest discussing the top-level questions first.